### PR TITLE
Release `1.0.0-beta.0`

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/client",
-  "version": "0.3.0",
```
Comment (krzysu) [inline packages/client/package.json:3]: the client should be also 1.0.0, no?

File: .changeset/pre.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "@lens-protocol/prettier-config": "0.5.0",
+    "@lens-protocol/react": "0.6.0",
+    "@lens-protocol/react-web": "0.6.0",
+    "@lens-protocol/sdk-gated": "1.2.0",
+    "@lens-protocol/shared-kernel": "0.6.0",
+    "@lens-protocol/storage": "0.6.0",
+    "@lens-protocol/tsconfig": "0.1.0",
+    "@lens-protocol/wagmi": "0.6.0"
+  },
+  "changesets": [
```
Comment (krzysu) [inline .changeset/pre.json:25]: what is this file for? haven't seen that before

File: packages/client/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.3.1-beta.0",
```
Comment (desfero) [inline packages/client/package.json:3]: should this one also be the `1.0.0`?

File: .changeset/pre.json (hunk 2 above)
Comment (cesarenaldi) [reply .changeset/pre.json:25]: Changeset feature: https://github.com/changesets/changesets/blob/main/docs/prereleases.md

**Review by krzysu — APPROVED** [review]

File: .changeset/pre.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "tag": "beta",
+  "initialVersions": {
+    "example-nextjs": "0.1.0",
+    "example-node": "0.0.1",
+    "example-react-native": "0.0.1",
+    "example-shared": "0.1.0",
+    "example-wagmi": "0.1.0",
+    "@lens-protocol/api-bindings": "0.6.0",
+    "@lens-protocol/blockchain-bindings": "0.6.0",
+    "@lens-protocol/client": "0.3.0",
```
Comment (krzysu) [inline .changeset/pre.json:12]: the client is still 0.3.0 here

**Comment by krzysu** [comment]
remember to squash all commits on this branch before releasing to npm, to have a single commit with the release tags

**Comment by cesarenaldi** [comment]
> remember to squash all commits on this branch before releasing to npm, to have a single commit with the release tags

thank you! I didn't think of this.

File: .changeset/pre.json (hunk 4 above)
Comment (cesarenaldi) [reply .changeset/pre.json:12]: these are all initial versions... before entering the pre-release mode.

Comment (krzysu) [reply .changeset/pre.json:12]: oh, I see, sorry!

**Pull request merged**